dockerfiles/web/Dockerfile

```diff
@@ -3,37 +3,69 @@
 ###################
 FROM python:3.8 as base
 
-RUN apt-get update && \
-    apt-get install -y \
-    python-openssl \
-    libpng-dev \
-    libjpeg-dev \
-    libjpeg62-turbo-dev \
-    libfreetype6-dev \
-    libxft-dev \
-    libffi-dev \
-    wget \
-    gettext \
-    libopenslide-dev \
-    libvips-dev
+# Build and install gdcm and install system dependencies
+RUN apt-get update \
+    && apt-get install -y \
+        cmake \
+        swig \
+        python-openssl \
+        libpng-dev \
+        libjpeg-dev \
+        libjpeg62-turbo-dev \
+        libfreetype6-dev \
+        libxft-dev \
+        libffi-dev \
+        wget \
+        gettext \
+        libopenslide-dev \
+        libvips-dev \
+    && mkdir -p /opt/gdcm/build \
+    && cd /opt/gdcm \
+    && git clone --branch v3.0.6 --depth 1 git://git.code.sf.net/p/gdcm/gdcm \
```

Comment:
Should we add an issue to switch to master when possible?

Reply:
I don't think so - it's best to pin our library dependencies. I could not work out if GDCMs master was always installable, but this is how they do releases. We could move it to a build arg, but, this is also where we update python so I don't know, maybe tidy it up when splitting the container builds from the Django tests as these both take ~15 minutes now.

Reply:
Ah yes of course, if this is their official release branch, then it would actually be better than using master.